### ISSUE TITLE
API Description not shown

### ISSUE THREAD

**Issue by kevindixon** [awslabs/aws-api-gateway-developer-portal#317]
API description is not shown in the UI for both API Gateway and imported APIs.
#192 appears to be resolved, but nonetheless descriptions for APIs are not shown.

By this I mean to API description, i.e. `info.description`:
```{
"swagger": "2.0",
"info": {
"description": "This is a sample server Petstore server.  You can find out more about Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For this sample, you can use the api key `special-key` to test the authorization filters.",
...```

**Comment by dylanbudgen** [comment]
Further to this, any API which is exported as swagger is missing the description field despite the description showing correctly in the gateway.

**Comment by gustakasn0v** [comment]
@a-tan In #192 you mentioned showing an API's description. I think you meant something like:
```
...
paths:
  /some/path
    get:
      summary: "Some summary."
      description: "Some long description"
      ...
```

Am I correct?

If so, it sounds like displaying the top-level API description (possibly right above the Endpoint) is still missing.

**Comment by amazon-meaisiah** [comment]
Fixed by #375 and released in v4.0.0.

**Comment by chrispward** [comment]
I dont know if its been re-introduced by description is not appearing in  the version I provisioned on thur via UI (7/7/22)
